packages/contracts/contracts/Passport.sol

```diff
@@ -11,24 +11,20 @@ contract Passport is ERC721, Ownable {
   address payable public _owner;
   mapping(uint256 => bool) public sold;
   uint256 public price;
+  uint256 public supply;
   event Purchase(address owner, uint256 price, uint256 id, string uri);
   constructor(string memory name_, string memory symbol_, uint256[] memory tokenIds, uint256 _price) ERC721(name_, symbol_) {
```

Comment:
decided to keep 721 over 1155 for now.

1155 allows you to have fungible amounts of each NFT - something that I think we probably want to strictly prohibit with passports